.golangci.yml

```diff
@@ -0,0 +1,8 @@
+linters:
```

Comment:
can you mirror the client settings: https://github.com/keybase/client/blob/master/go/.golangci.yml#L5-L15

Reply:
Mirrored except stricter. There are some additional linters that I have enabled that seem useful and are more practical to have enabled given that this is a smaller project. Do you object to any of the additional ones I added?

Reply:
sounds great!